Localizations/en_US.json

```diff
@@ -89,8 +89,8 @@
     "home": "Home",
     "ignore_engine_assets": "Ignore Engine Assets",
     "image_export_title": "Image Export Format",
-    "image_format_description": "Affects all exported images",
-    "json": "Json",
+    "image_format_description": "Affects all exported images.",
```

Comment:
In hindsight, this should have been a full sentence.

> This affects all exported images.

Reply:
Ah, okay! I'll fix that!